python/MetrologyData.py

```diff
@@ -342,29 +274,6 @@ def plot_statistics(self, nsigma=5, title=None, zoffset=0):
         y = y/np.sum(y)*np.size(dz)*binsz/0.1
         plot.pylab.plot(x, y, color='b', linestyle='-')
 
-        # Overlay the model residuals
```

Comment:
I was using round( ) so that the x-axis range would have integer limits, but that really isn't important, and you are right about the danger of rounding to zero.  I'll remove the round( )